src/App.tsx

```diff
@@ -1,24 +1,74 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './components/Clock';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+type State = {
+  isClockVisible: boolean;
+  clockName: number;
 };
 
+class App extends React.Component<{}, State> {
+  state = {
+    isClockVisible: true,
+    clockName: 1,
+  };
+
+  toggleClockVisibility(bool: boolean) {
```

Comment:
bool name doesn’t describe for what this value

```suggestion
  toggleClockVisibility(isVisible: boolean) {
```